p2p/src/main/java/com/google/android/fhir/p2p/SyncPayload.kt

```diff
@@ -16,22 +16,17 @@
 
 package com.google.android.fhir.p2p
 
-import ca.uhn.fhir.context.FhirContext
-import kotlinx.serialization.KSerializer
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
-import kotlinx.serialization.descriptors.PrimitiveKind
-import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
-import kotlinx.serialization.descriptors.SerialDescriptor
-import kotlinx.serialization.encoding.Decoder
-import kotlinx.serialization.encoding.Encoder
-import org.hl7.fhir.instance.model.api.IBaseResource
+import org.hl7.fhir.r4.model.ResourceType
 
 /** The payload for syncing changes between peers. */
 @Serializable
 data class SyncPayload(
-  /** The current version of the resource after all of the changes have been applied. */
-  @Serializable(with = FhirResourceSerializer::class) val resource: IBaseResource,
+  val resourceId: String,
+  val resourceType: ResourceType,
+  val lastUpdatedDate: String,
```

Comment:
Using the term "Date" is incorrect here. It should be "Timestamp".

Reply:
I was trying to be consistent with the now of the property on the entities but happy to change.

```kotlin
@Entity
data class SyncedResourceEntity(
  /** Resource synced */
  @PrimaryKey val resourceType: ResourceType,
  /** The highest `_lastUpdate` value of the resources synced of a specific type */
  val lastUpdate: String
)
```

Reply:
I'm not sure how that's consistent. Should we not just be calling it `lastUpdated` then?

Reply:
Renamed to `lastUpdateTimestamp`.